corehq/apps/sms/filters.py

```diff
@@ -71,9 +71,9 @@ class EventStatusFilter(BaseSingleOptionFilter):
     options = STATUS_CHOICES
 
 
-class PhoneNumberFilter(SearchFilter):
+class PhoneNumberFilter(BaseSimpleFilter):
     label = ugettext_lazy("Phone Number")
-    search_help_inline = ugettext_lazy("Enter a full or partial phone number to filter results")
+    help_inline = ugettext_lazy("Enter a full or partial phone number to filter results")
```

Comment:
Does this one need a `slug`? `SearchFilter.slug` was `"search_query"` but `BaseSimpleFilter.slug` is `None`.

Reply:
Thanks, updated: 70dd0a3